### PR TITLE
How to [Doc]: Create python 3.7 virtual env

### PR DESCRIPTION
**Pull request by eladiw**

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -44,9 +51,9 @@
     $ pip3 install cython
     ```
 
-7. Install [tesseract](https://github.com/tesseract-ocr/tesseract/wiki) OCR framework.
+8. Install [tesseract](https://github.com/tesseract-ocr/tesseract/wiki) OCR framework.
```
Comment (omri374) [inline docs/development.md]: optional or mandatory?

Comment (eladiw) [reply docs/development.md]: Depends, if wants to work on OCR... then mandatory

File: docs/development.md [hunk 2]
```diff
@@ -31,7 +31,14 @@
     $ cd re2 && make install
     ```
 
-6. Install the Python packages for the analyzer in the `presidio-analyzer` folder
+6. Create a Python virtual environment
+
+    ```sh
+    python3.6 -m venv presidio-venv
```
Comment (eladiw) [inline docs/development.md]:
```suggestion
    python3.7 -m venv presidio-venv
```

File: docs/development.md (hunk 1 above)
Comment (omri374) [reply docs/development.md]: So it's optional :)

File: docs/development.md [hunk 3]
```diff
@@ -31,7 +31,14 @@
...
     ```
 
-6. Install the Python packages for the analyzer in the `presidio-analyzer` folder
+6. Create a Python virtual environment
+
+    ```sh
+    python3.7 -m venv presidio-venv
+    source presidio-venv/bin/activate
+    ```
+
```
Comment (omri374) [inline docs/development.md]: I'll test this manually before approving

**Comment by omri374** [comment]
Moving to pipenv

**Pull request closed without merging**